api/src/main/java/me/lokka30/treasury/api/economy/account/Account.java

```diff
@@ -73,6 +73,15 @@ public interface Account {
     @NotNull
     CompletableFuture<Response<BigDecimal>> retrieveBalance(@NotNull Currency currency);
 
+    /**
+     * Gets the starting balance of this {@code Account} for the specified {@link Currency currency}
```

Comment:
```suggestion
     * Gets the starting balance of this account for the specified {@link Currency currency}.
```

Reply:
`{@code Account}` looks better. prove me wrong

Reply:
It's not code, neither it's not a specific key to look for in the config. Would you prefer:

```java
{@link Account account}
```

?

Reply:
![img](https://img.mrivanplays.com/ycuqczpdxh.png)

See the difference? It gets a different font.